js/main.js

```diff
@@ -0,0 +1,87 @@
+const DESCRIPTION = [
+  'Моё фото',
+  'Наслаждаюсь фотографией',
+  'Красота!',
+  'Все быстренько ставим лайки!',
+  'Кайфую от этой фотографии',
+  'Как вам фотокарточка?',
+  'Photo by me',
+];
+
+const MESSAGE = [
+  'Всё отлично!',
+  'В целом всё неплохо. Но не всё.',
+  'Когда вы делаете фотографию, хорошо бы убирать палец из кадра. В конце концов это просто непрофессионально.',
+  'Моя бабушка случайно чихнула с фотоаппаратом в руках и у неё получилась фотография лучше.',
+  'Я поскользнулся на банановой кожуре и уронил фотоаппарат на кота и у меня получилась фотография лучше.',
+  'Лица у людей на фотке перекошены, как будто их избивают. Как можно было поймать такой неудачный момент?!',
+];
+
+const NAME = [
+  'Тинки-Винки',
+  'Дипси',
+  'Ляля',
+  'По',
+];
+
+const SIMILAR_PHOTO_DESCRIPTION_COUNT = 25;
+
+const getRandomInteger = (min, max) => {
+  const lower = Math.ceil(Math.min(Math.abs(min), Math.abs(max)));
+  const upper = Math.floor(Math.max(Math.abs(min), Math.abs(max)));
+  const result = Math.random() * (upper - lower + 1) + lower;
+  return Math.floor(result);
+};
+
+const getRandomArrayElement = (elements) => elements[getRandomInteger(0, elements.length - 1)];
+
+const createRandomId = (min, max) => {
+  const previousValues = [];
+  return function () {
+    let currentValue = getRandomInteger(min, max);
+    if (previousValues.length >= (max - min + 1)) {
```

Comment:
Доп критерий про единообразие функций
```suggestion
  return () => {
```